R/tm_data_table.R

```diff
@@ -318,9 +318,12 @@ srv_data_table <- function(id,
       df <- data()[[dataname]]
 
       teal::validate_has_data(df, min_nrow = 1L, msg = paste("data", dataname, "is empty"))
-
-      teal.code::eval_code(
+      qenv <- teal.code::eval_code(
         data(),
+        'library("ggplot2");library("dplyr");library("DT")'
```

Comment:
I don't think there is `ggplot2` in DT : P
```suggestion
        'library("dplyr");library("DT")'
```